tools/codegen/gen.py

```diff
@@ -263,7 +263,7 @@ def __call__(self, f: NativeFunction) -> str:
 STATIC_CONST_STR_OUT_OF_LINE_FOR_WIN_CUDA({name}, schema_str, {cpp_string(str(f.func))})
 
 // aten::{f.func}
-static C10_NOINLINE c10::TypedOperatorHandle<{name}::schema> create_{name}_typed_handle() {{
+static c10::TypedOperatorHandle<{name}::schema> create_{name}_typed_handle() {{
```

Comment:
Funnily enough, the `C10_NOINLINE` is meant to _reduce_ binary size here. The issue I've found is that this has changed the inlining decisions, so more of the `OperatorEntry` code is being inlined. I've got a fix for this and will open a PR soon.

Reply:
What would be the ETA for that PR?

Reply:
@albanD it's linked below, see  #63127.